flyp.php

```diff
@@ -50,6 +50,11 @@ function bnomics_create_order(){
     	$blockonomics = new Blockonomics();
     	$whmcs_order_id = $blockonomics->getOrderIdByInvoiceId($whmcs_invoice_id);
     	$blockonomics->updateFlypIdInDb($whmcs_order_id, $order->order->uuid);
+        $actual_link = $blockonomics->getSystemUrl();
+        $invoiceNote = "<b>Waiting for Confirmation on $flypFrom network</b>\r\r" .
```

Comment:
Only say FLY UUID: <value>

No need for links/Waiting for confirmation. Merchant shouldn't claim the refund and also we need say Waiting for confirmation, as it may have confirmed

Reply:
42dd751 - Updated invoice note to only show `FLYP UUID: value`

*This invoice note will get replaced when the BTC transaction is received in the callback